src/string_map.cpp

```diff
@@ -25,295 +25,6 @@ gb_internal gb_inline StringHashKey string_hash_string(String const &s) {
 }
 
 
-#if 1 /* old string map */
```

Comment:
Might be better to just set this to `0` rather than delete the code

Reply:
Done.